docs/getting-started/croeseid-testnet.md

```diff
@@ -115,12 +115,12 @@ Follow the below optional steps to enable state-sync.
 - (Optional)For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
 
   ```bash
-  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid-1.crypto.com:26657/block | jq -r .result.block.header.height); \
+  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.com:26657/block | jq -r .result.block.header.height); \
```

Comment:
We need the new chain to run for a while before we can have at least 1000 blocks.

Reply:
Yes we do, no worries 😉